scipy/interpolate/fitpack.py

```diff
@@ -273,16 +273,7 @@ def splrep(x, y, w=None, xb=None, xe=None, k=3, task=0, s=None, t=None,
 
     Examples
     --------
-
-    >>> import matplotlib.pyplot as plt
-    >>> from scipy.interpolate import splev, splrep
-    >>> x = np.linspace(0, 10, 10)
-    >>> y = np.sin(x)
-    >>> spl = splrep(x, y)
-    >>> x2 = np.linspace(0, 10, 200)
-    >>> y2 = splev(x2, spl)
-    >>> plt.plot(x, y, 'o', x2, y2)
-    >>> plt.show()
+    Examples are given :ref:`in the tutorial <tutorial-interpolate_splXXX>`.
```

Comment:
Can you leave the example which was there? A link to the turorial is good, maybe "further examples are given in..."

Reply:
Thank you for reviewing. I fixed it.